.github/workflows/shared-terratest-queue.yml

```
@@ -0,0 +1,484 @@
+      - name: "Check if test/run.sh exists"
+        id: skip-tests
+        run: |-
+          if [ -f "test/run.sh" ]; then
+            echo "result=true" >> $GITHUB_OUTPUT
+          else
+            echo "result=false" >> $GITHUB_OUTPUT
+          fi  
+
+      - name: "Comment on PR if tests skipped"
+        if: ${{ steps.skip-tests.outputs.result == 'true' }}
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)